### PR TITLE
Update verifyUser and add embedMessage

### PR DESCRIPTION
**Pull request by AndrewJWin**
This PR updates the verifyUser method to properly throw errors in the event that "message" or "user" is not fulfilled.
This PR also includes the new embedMessage method, this method aims to simplify the duplication of default code within multiple commands, specifically the footer, color and timestamp. 

Original verifyUser method.
verifyUser1

Updated, missing ``Message`` and ``User``.
verifyUser2

Missing ``User``.
verifyUser3

Proper usage.
verifyUser4

* embedMessage method.

Invalid use of, missing ``Message``.
embedMessage1

Proper usage.
embedMessage2

**Semantic versioning classification:**

- [x] This PR changes the framework's interface (methods or parameters added)
- [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

File: base/Command.js [hunk 1]
```diff
@@ -64,14 +64,21 @@ class Command {
     return `${line1}${line2}`;
   }
 
+  async embedMessage(message) {
+    if (!message) throw new this.client.methods.errors.ParseError("Invalid format, missing required value: MESSAGE");
+    const embed = await new MessageEmbed().setColor(message.guild.me.roles.highest.color || 5198940).setTimestamp().setFooter(`Requested by ${message.member.user.username}`, message.guild.me.user.displayAvatarURL());
```
Comment (fratik) [inline base/Command.js:69]: why await?

Comment (AndrewJWin) [reply base/Command.js:69]: You have to create the embed...

Comment (DevYukine) [reply base/Command.js:69]: This is pointless since JS doesn't have async constructors aswell as non of the methods return a Promise

**Review by DevYukine — CHANGES_REQUESTED** [review]

File: base/Command.js (hunk 1 above)
Comment (fratik) [reply base/Command.js:69]: ^ exactly

Comment (MrJacz) [inline base/Command.js:69]: This line is very long please add some new lines.

File: base/Command.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { Permissions } = require("discord.js");
+const { Permissions, MessageEmbed } = require("discord.js");
```
Comment (MrJacz) [inline base/Command.js:1]: you can use `this.client.methods.Embed`

**Review by MrJacz — CHANGES_REQUESTED** [review]

**Comment by AndrewJWin** [comment]
**Going to redo and re-open at a later time while fixing Conflicts.

**Pull request closed without merging**